README.md

```diff
@@ -66,6 +66,7 @@ Add a new entry to the `hierarchy` hash in `hiera.yaml` referencing the vault na
       vault_name: production-vault
       vault_api_version: '2016-10-01'
       metadata_api_version: '2018-04-02'
+      key_replacement_token: '-'
```

Comment:
Can we add some documentation in the readme.md that helps developers understand what this is doing, why it's needed, and how this will affect how they name objects in keyvault?

Reply:
Done.